internal/controllers/instance_controller.go

```diff
@@ -141,7 +142,8 @@ func (r *instanceReconciler) reconcileObjects(ctx context.Context, inst cosmov1a
 			continue
 		}
 
-		if mapping.Scope != inst.GetScope() {
+		// nanespaced scope instance cannot create cluster scope resources
```

Comment:
typo "nanespaced"

Reply:
I will fix next PR.

Reply:
It is not typo.
Object in namespaces is called "namespaced object" in Kubernetes.

https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/#not-all-objects-are-in-a-namespace

```
kubectl api-resources --namespaced=true
```